lib/view_model/transaction_details_view_model.dart

```diff
@@ -146,6 +149,7 @@ abstract class TransactionDetailsViewModelBase with Store {
       case WalletType.monero:
         return S.current.view_transaction_on + 'Monero.com';
       case WalletType.bitcoin:
+      case WalletType.lightning:
```

Comment:
didn't you say it doesn't have an explorer, so you should add a case to exclude this from lightning